platform_disk_api/service.py

```diff
@@ -186,10 +193,13 @@ async def get_disk_by_name(self, name: str, owner: str) -> Disk:
             raise DiskNotFound
         return await self._pvc_to_disk(pvc)
 
-    async def get_all_disks(self) -> list[Disk]:
+    async def get_all_disks(self, project_name: Optional[str] = None) -> list[Disk]:
+        label_selector = None
+        if project_name:
+            label_selector = f"{PROJECT_LABEL}={project_name}"
         return [
             await self._pvc_to_disk(pvc)
-            for pvc in await self._kube_client.list_pvc()
+            for pvc in await self._kube_client.list_pvc(label_selector)
```

Comment:
Tenants (orgs within clusters, or no org within cluster) can have projects with the same name.
I think we need to stick to the traversal logic in `self._check_disk_read_perm` instead of relying on the labelSelector here. wdyt?

Reply:
didn't get you. label selector will remove data not related to project_name from response and then pass response to traversal logic.

Reply:
yes, it makes sense to support also org_name query parameter in the endpoint, because as far as I understand org_name filtering is currently done on client side.

Reply:
added org_name parameter